modules/@apostrophecms/schema/ui/apos/components/AposSearchList.vue

```diff
@@ -32,16 +33,37 @@ export default {
       default() {
         return [];
       }
+    },
+    disabledTooltip: {
+      type: String,
+      required: false
     }
   },
   emits: [ 'select' ],
   methods: {
-    select(item) {
+    select(item, $event) {
+      if (item.disabled) {
+        $event.stopPropagation();
+        return;
+      }
       const selectedItems = this.selectedItems;
       if (!selectedItems.some(selectedItem => selectedItem._id === item._id)) {
         // Never modify a prop
         this.$emit('select', [ ...selectedItems, item ]);
       }
+    },
+    getClasses(item) {
+      const classes = {
+        'apos-search__item': true,
+        'item': true
```

Comment:
what is this, what is being styled with this selector? What can't `apos-search__item` do?

also, all classes have to be prefixed `apos-`.